assets/blocks/computer.block

```diff
@@ -0,0 +1,13 @@
+{
+    "displayName": "computer",
+    "categories" : ["computer"],
+    "hardness": 7,
+    "tiles" : {
```

Comment:
You can improve this a bit more by adding a rotation to the block.
Have a look at the [Chest](https://github.com/MovingBlocks/Terasology/blob/develop/modules/Core/assets/blocks/Chest.block) for an example.

`"rotation": "horizontal",` tells the system to create a copy of your block, with the front facing in each horizontal direction. When placing the block, the block with the front facing you is placed.
This behavior is comparable to a minecraft furnance: It will always face you but you can't place it sideways so that the front looks upwards.